api/v1/constructors.go

```diff
@@ -935,8 +935,9 @@ func parseCertificateInfo(spec *SystemSpec, certificates []certificates.Certific
 	result := make([]CertificateInfo, 0)
 
 	for index, c := range certificates {
-		// Ignore OpenLDAP certificate since it's being installed during the initial unlock.
-		if c.Type == "openldap" {
+		// Ignore the OpenLDAP/Docker/Platform certificates installed during bootstrap/initial unlock
+		if c.Type == OpenLDAPCertificate || c.Type == DockerCertificate ||
+			c.Type == PlatformCertificate {
```

Comment:
What about ignoring `OpenstackCACertificate `?

Reply:
I'm not sure about this one, the UT question would depend on the change here. I'm currently addressing ssl/docker_registry that will afect DM after a change in stx master.
@yjian118 do you know how are we supposed to deal with openstack_ca if included by the user in the DM config file?

Reply:
I didn't modify this since my changes are focused in ssl/docker_registry, that will be present and curated by stx from bootstrap.
I believe we are filtering it out from the output if the user builds the config file from the system. I'll include the code to filter these as well from the provided spec.